packages/react-app/src/views/YourLoogies.jsx

```diff
@@ -18,45 +18,52 @@ function YourLoogies({
   transferToAddresses,
   setTransferToAddresses,
   address,
+  isLoading,
 }) {
   return (
     <div className="your-loogies">
+      <h2 className="your-loogies__title">Sneak peak on your Loogies with a smile:</h2>
+      <p className="your-loogies__description">The more loogies the happier they are!</p>
       <div style={{ margin: "auto", paddingBottom: 25 }}>
-        <List
-          grid={{
-            gutter: 16,
-            xs: 1,
-            sm: 2,
-            md: 2,
-            lg: 3,
-            xl: 4,
-            xxl: 6,
-          }}
-          dataSource={yourCollectibles}
-          renderItem={item => {
-            const id = item.id.toNumber();
+        {isLoading ? (
```

Comment:
Yo can remove the conditional and add a "loading" attribute to the List compoenent:

loading={isLoading}